api/src/main/java/org/openmrs/module/openconceptlab/client/OclMapping.java

```diff
@@ -126,7 +126,7 @@ public void setFromConceptUrl(String fromConceptUrl) {
 	}
 	
 	public String getToSourceName() {
-		return toSourceName;
+		return toSourceName != null ? toSourceName : toSourceNameResolved;

```

Comment:
Thanks @jwamalwa ,could you first declare the variable toSourceNameResolved before using it around here?